src/rpdk/core/contract/suite/handler_delete.py

```diff
@@ -35,15 +35,20 @@ def deleted_resource(resource_client):
         _status, response, _error = resource_client.call_and_assert(
             Action.DELETE, OperationStatus.SUCCESS, model
         )
-        assert "resourceModel" not in response
+        assert (
+            "resourceModel" not in response
+        ), "ProgressEvent object MUST NOT contain a model"
```

Comment:
Maybe say instead: "The deletion handler's response must not contain a model"

Just to keep it more in line with the other messages since we don't usually refer to the response as the ProgressEvent